aviary/interface/default_phase_info/height_energy.py

```diff
@@ -17,188 +15,85 @@
-            'required_available_climb_rate': (1.524, 'm/s'),
-            'mass_f_cruise': (1.e4, 'kg'),
-            'distance_f_cruise': (1.e6, 'm'),
-        },
-        'initial_guesses': {
-            'times': ([26.2, 406.18], 'min'),
-            'altitude': ([35.e3, 35.e3], 'ft'),
-            'velocity': ([455.49, 455.49], 'kn'),
-            'mass': ([165.e3, 140.e3], 'lbm'),
-            'distance': ([160.3, 3243.9], 'nmi'),
-            'velocity_rate': ([0., 0.], 'm/s**2'),
-            'throttle': ([0.95, 0.9], 'unitless'),
-        }
+        "initial_guesses": {"times": ([128, 113], "min")},
     },
-    'descent': {
-        'subsystem_options': {
-            'core_aerodynamics': {'method': 'computed'}
-        },
-        'user_options': {
-            'fix_initial': False,
-            'fix_range': True,
-            'fix_duration': False,
-            'num_segments': 5,
-            'order': 3,
-            'initial_ref': (1., 's'),
-            'initial_bounds': ((10.e3, 30.e3), 's'),
-            'duration_ref': (1754.4, 's'),
-            'duration_bounds': ((726., 3508.8), 's'),
-            'initial_altitude': (10.668e3, 'm'),
-            'final_altitude': (10.668, 'm'),
-            'initial_mach': 0.79,
-            'final_mach': 0.3,
-            'no_climb': False,
+    "descent": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.36, "unitless"),
+            "mach_bounds": ((0.34, 0.74), "unitless"),
+            "initial_altitude": (34000.0, "ft"),
+            "final_altitude": (500.0, "ft"),
+            "altitude_bounds": ((0.0, 38000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": False,
+            "constrain_final": True,
+            "fix_duration": False,
+            "initial_bounds": ((120.5, 361.5), "min"),
+            "duration_bounds": ((29.0, 87.0), "min"),
         },
-        'initial_guesses': {
-            'times': ([432.38, 29.24], 'min'),
-            'altitude': ([35.e3, 35.], 'ft'),
-            'velocity': ([455.49, 198.44], 'kn'),
-            'mass': ([120.e3, 115.e3], 'lbm'),
-            'distance': ([3243.9, 3378.7], 'nmi'),
-            'velocity_rate': ([-0.25, 0.0], 'm/s**2'),
-            'throttle': ([0., 0.], 'unitless'),
-        }
+        "initial_guesses": {"times": ([241, 58], "min")},
     },
-    'post_mission': {
-        'include_landing': True,
+    "post_mission": {
+        "include_landing": False,
+        "constrain_range": True,
+        "target_range": (1906, "nmi"),
     },
 }
-
-
-def phase_info_parameterization(phase_info, aviary_inputs):
```

Comment:
As in earlier comment, this needs to be migrated to not lose a command line feature that was essential to the release.

Reply:
Agreed, added back now.